README.md

```diff
@@ -64,19 +84,58 @@ Apply all the following sections in sequence to configure and install the connec
    cp .env.sample .env
    ```
 
+### Choosing the platform
+
+Open the `.env` file and choose between GCP or AWS.
+
+#### Using AWS
+
+AWS is the default.
+
+- Edit the `CTP_POST_BODY` variable under `For AWS`: It contains the commercetools API Extension configuration parameters.
+  - `accessKey`: An AWS access key.
+  - `accessSecret`: An AWS secret.
+
+To configure `accessKey` and `accessSecret` create another IAM user with minimal
+privileges. See an example in [`iam/ct-dev-iam.sample.json`](iam/ct-dev-iam.sample.json).
+Also see [AWS Lambda](https://docs.commercetools.com/api/projects/api-extensions#aws-lambda-destination).
+
+#### Using GCP
+
+Uncomment all commented variables under `For Google` and edit them:
+
+- `CTP_POST_BODY`: It contains the commercetools API Extension configuration parameters.
+
+  - `url`: URL for the lambda function. Retrieve it from CLI after deployment or from Google Console.
+    Example: `https://europe-west3-t1-integration.cloudfunctions.net/t1-ct-dev-api-extension`
+  - `headerValue`: A combination of `BASIC_AUTH_USERNAME` and `BASIC_AUTH_PASSWORD`. You can generate a hash with `echo -n "<user>:<password>" | base64`.
+
+- `PROVIDER`: `google`. Defaults to: `aws`.
+- `GCP_PROJECT`: Google project name.
+- `GCP_CREDENTIALS`: The absolute path to your Google service account credentials file created in the [Installing the tools section](#installing-the-tools).
+- `BASIC_AUTH_USERNAME` and `BASIC_AUTH_PASSWORD`: for security reasons we need to set
+  up basic authentication for the HTTP endpoint (same as in the first point)
+
+##### Example
+
+```ini
+CTP_POST_BODY="{"destination":{"type":"HTTP","url":"https://europe-west3-t1-integration.cloudfunctions.net/t1-ct-dev-api-extension","authentication":{"type":"AuthorizationHeader","headerValue":"Basic dXNlcjEyMzpwYXNzd29yZDEyMw=="}},"triggers":[{"resourceTypeId":"cart","actions":["Create","Update"]},{"resourceTypeId":"customer","actions":["Create","Update"]},{"resourceTypeId":"order","actions":["Create","Update"]}]}"
```

Comment:
@ArthurFlageul 
What do you think of attaching also a json-highlighted separate code block with the contents of this env variable, to help the reader understand what goes in there 🤔 

```json
{
  "destination": {
    "type": "HTTP",
    "url": "https://europe-west3-t1-integration.cloudfunctions.net/t1-ct-dev-api-extension",
    "authentication": {
      "type": "AuthorizationHeader",
      "headerValue": "Basic dXNlcjEyMzpwYXNzd29yZDEyMw=="
    }
  },
  "triggers": [
    { "resourceTypeId": "cart", "actions": ["Create", "Update"] },
    { "resourceTypeId": "customer", "actions": ["Create", "Update"] },
    { "resourceTypeId": "order", "actions": ["Create", "Update"] }
  ]
}
```